docs/tutorials/operations.md

```diff
@@ -0,0 +1,166 @@
+instead. Path prefix search is much more scalable and can not lead to
+unexpected results by not so experienced regular expressions users.
+
+To find more information about Metrics, including formats and example
+Prometheus queries you find in the [metrics
+section](../../operation/operation.md#monitoring).
+The settings shown above support system and application metrics to
+carefully monitor Skipper and your backend applications. Backend
+application metrics get error rates and latency buckets based on host
+headers. The chosen options are a good setup to safely run all
+workloads from small to high traffic.
+
+The option `-max-audit-body=0`, won't log the HTTP body, if you would
+do audit logging, to have a safe default.
+
+The last option `-idle-timeout-server=62s` was chosen, because of a
+[known issue](https://github.com/zalando/skipper/issues/964), if you
+run in a multi layer loadbalancer, with ALBs in front of Skipper.
+[ALBs idle connection timeout is 60s](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/application-load-balancers.html#connection-idle-timeout)
+and AWS support told us to run the backends with a bigger timeout,
+than the ALB in front.
+
+### Opt-In more features
+
+Depending on the HTTP loadbalancer in front of your Skippers, you might
+want to set `-reverse-source-predicate`. This setting reverse the
+lookup of the client IP to find it in the `X-Forwarded-For` header
+values. If you do not care about
+[clientRatelimits](../../reference/filters.md#clientratelimit)
+based on X-Forwarded-For headers, you can also ignore this.
+
+Ratelimits can be calculated for the whole cluster instead of having
+only the instance based ratelimits. The common term we use in skipper
+documentation is [cluster ratelimit](https://opensource.zalando.com/skipper/tutorials/ratelimit/#cluster-ratelimit).
+There are two option, but we highly recommend the use of Redis based
+cluster ratelimits. To support redis based cluster ratelimits you have to
+use `-enable-swarm` and add a list of URLs to redis
+`-swarm-redis-urls=skipper-ingress-redis-0.skipper-ingress-redis.kube-system.svc.cluster.local:6379,skipper-ingress-redis-1.skipper-ingress-redis.kube-system.svc.cluster.local:6379`. We
+run [redis as
+statefulset](https://github.com/zalando-incubator/kubernetes-on-aws/blob/beta/cluster/manifests/skipper/skipper-redis.yaml)
+with a [headless
+service](https://github.com/zalando-incubator/kubernetes-on-aws/blob/beta/cluster/manifests/skipper/skipper-redis-service.yaml)
+to have predictable names. We chose to not use a persistent volume,
+because storing the data in memory is good enough for this use case.
+
+Skipper supports cluster internal service to service communication as
+part of running as an [API Gateway with an East-West
+setup](../../kubernetes/ingress-controller/#run-as-api-gateway-with-east-west-setup).
+You have to add `-enable-kubernetes-east-west` and optional choose the
+domain of your choice
```

Comment:
"...choose the domain of your choice" -> "...choose a domain"

Reply:
fixed